2025-03-13 18:48:59,415 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_fz7oIQ2goRLfrP1mWceBcjje "HTTP/1.1 200 OK"
2025-03-13 18:49:00,438 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads "HTTP/1.1 200 OK"
2025-03-13 18:49:00,921 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/messages "HTTP/1.1 200 OK"
2025-03-13 18:49:02,209 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs "HTTP/1.1 200 OK"
2025-03-13 18:49:02,692 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:04,254 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:05,705 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:07,315 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:08,891 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:10,576 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:12,042 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:13,564 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:15,087 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:16,801 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/runs/run_c0Pl0FxmQSr1GZkOe72UdH2g "HTTP/1.1 200 OK"
2025-03-13 18:49:17,235 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_U2cKf7u3lKMPYbHRrO3GeEFG/messages "HTTP/1.1 200 OK"
2025-03-13 18:49:17,441 - app.api.routes.thread - ERROR - Callback failed: HTTPSConnectionPool(host='api.glific.test', port=4001): Max retries exceeded with url: /webhook/flow_resume (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))
2025-03-13 19:40:23,657 - watchfiles.main - INFO - 1 change detected
2025-03-13 19:40:24,253 - watchfiles.main - INFO - 2 changes detected
2025-03-13 19:40:25,219 - watchfiles.main - INFO - 3 changes detected
2025-03-13 19:40:28,172 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_fz7oIQ2goRLfrP1mWceBcjje "HTTP/1.1 200 OK"
2025-03-13 19:40:29,080 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads "HTTP/1.1 200 OK"
2025-03-13 19:40:55,265 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/messages "HTTP/1.1 200 OK"
2025-03-13 19:40:57,559 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs "HTTP/1.1 200 OK"
2025-03-13 19:40:58,131 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:40:59,717 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:01,255 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:02,739 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:04,216 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:05,767 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:07,936 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:09,425 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:10,928 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:12,472 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:14,208 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:15,939 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:17,788 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:19,325 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:20,782 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:22,466 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:23,937 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/runs/run_0ADGv5FOBf1jZru1WqlG9mfX "HTTP/1.1 200 OK"
2025-03-13 19:41:24,367 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_iiti8VmmLPLpNkvYMLrHfhvM/messages "HTTP/1.1 200 OK"
2025-03-13 19:41:24,406 - app.api.routes.thread - ERROR - Callback failed: HTTPSConnectionPool(host='api.glific.test', port=4001): Max retries exceeded with url: /webhook/flow_resume (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))
2025-03-13 19:41:25,023 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_fz7oIQ2goRLfrP1mWceBcjje "HTTP/1.1 200 OK"
2025-03-13 19:41:25,479 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads "HTTP/1.1 200 OK"
2025-03-13 19:41:26,019 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/messages "HTTP/1.1 200 OK"
2025-03-13 19:41:27,601 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs "HTTP/1.1 200 OK"
2025-03-13 19:41:28,308 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:29,821 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:31,323 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:32,920 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:34,721 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:36,196 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:37,708 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:39,651 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:42,255 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:45,673 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/runs/run_FwImHTrDrlEzmnBkYewDvCXA "HTTP/1.1 200 OK"
2025-03-13 19:41:46,056 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_bbqypMj8woCQg8Ibixx05HyO/messages "HTTP/1.1 200 OK"
2025-03-13 19:41:46,069 - app.api.routes.thread - ERROR - Callback failed: HTTPSConnectionPool(host='api.glific.test', port=4001): Max retries exceeded with url: /webhook/flow_resume (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))
2025-03-13 19:43:41,483 - watchfiles.main - INFO - 1 change detected
2025-03-13 19:43:41,992 - watchfiles.main - INFO - 1 change detected
2025-03-13 19:43:43,047 - watchfiles.main - INFO - 3 changes detected
2025-03-13 19:44:00,239 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_fz7oIQ2goRLfrP1mWceBcjje "HTTP/1.1 200 OK"
2025-03-13 19:44:00,875 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads "HTTP/1.1 200 OK"
2025-03-13 19:44:01,488 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/messages "HTTP/1.1 200 OK"
2025-03-13 19:44:02,752 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs "HTTP/1.1 200 OK"
2025-03-13 19:44:03,430 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 19:44:05,679 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 19:44:07,221 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 19:44:08,732 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 19:44:10,314 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 19:44:13,721 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 19:44:15,393 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 19:44:17,013 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 19:44:18,495 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 20:57:13,353 - openai._base_client - INFO - Retrying request to /threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t in 0.419063 seconds
2025-03-13 20:57:14,704 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/runs/run_Ofilr2Td1PFdLLSv3S3dxC7t "HTTP/1.1 200 OK"
2025-03-13 20:57:15,464 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_obWD1wxACnNuhKX0cz69rbu4/messages "HTTP/1.1 200 OK"
2025-03-13 20:57:15,524 - app.api.routes.thread - ERROR - Callback failed: HTTPSConnectionPool(host='api.glific.test', port=4001): Max retries exceeded with url: /webhook/flow_resume (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))
2025-03-17 13:03:26,870 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_fz7oIQ2goRLfrP1mWceBcjje "HTTP/1.1 200 OK"
2025-03-17 13:03:27,323 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads "HTTP/1.1 200 OK"
2025-03-17 13:03:27,816 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/messages "HTTP/1.1 200 OK"
2025-03-17 13:03:28,818 - httpx - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/runs "HTTP/1.1 200 OK"
2025-03-17 13:03:29,296 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/runs/run_B8cxe2raQWtpN46Hzd2xUNzK "HTTP/1.1 200 OK"
2025-03-17 13:03:30,803 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/runs/run_B8cxe2raQWtpN46Hzd2xUNzK "HTTP/1.1 200 OK"
2025-03-17 13:03:32,378 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/runs/run_B8cxe2raQWtpN46Hzd2xUNzK "HTTP/1.1 200 OK"
2025-03-17 13:03:33,841 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/runs/run_B8cxe2raQWtpN46Hzd2xUNzK "HTTP/1.1 200 OK"
2025-03-17 13:03:35,364 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/runs/run_B8cxe2raQWtpN46Hzd2xUNzK "HTTP/1.1 200 OK"
2025-03-17 13:03:36,827 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/runs/run_B8cxe2raQWtpN46Hzd2xUNzK "HTTP/1.1 200 OK"
2025-03-17 13:03:37,230 - httpx - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_qln1ScoogAeQCNZPzePM0VMz/messages "HTTP/1.1 200 OK"
2025-03-17 13:03:37,250 - app.api.routes.thread - ERROR - Callback failed: HTTPSConnectionPool(host='api.glific.test', port=4001): Max retries exceeded with url: /webhook/flow_resume (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))
